src/main/resources/js/condec.dialog.js

```diff
@@ -61,6 +61,74 @@
 		// Show dialog
 		AJS.dialog2(createDialog).show();
 	};
+	
+	ConDecDialog.prototype.showAssignDialog = function showAssignDialog(sourceId, documentationLocation) {
+		console.log("conDecDialog showAssignDialog");
+		
+		// HTML elements
+		var assignDialog = document.getElementById("assign-dialog");
+		var selectLevelField = document.getElementById("assign-form-select-level");
+		var example = document.getElementById("example-tag");
+		var inputExistingGroupsField = document.getElementById("assign-form-input-existing");
+		var inputAddGroupField = document.getElementById("assign-form-input-add");
+		var submitButton = document.getElementById("assign-dialog-submit-button");
+		var cancelButton = document.getElementById("assign-dialog-cancel-button");
+		
+		//Fill HTML elements
+		
+		conDecAPI.getDecisionGroups(sourceId,documentationLocation,inputExistingGroupsField,selectLevelField,function(selectLevelField, inputExistingGroupsField, groups){
+			if(!(groups === null) && groups.length > 0){
+				var groupZero = groups[0];
+				if("High_Level" == groupZero){

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5160185)